docs/application/native/guides/index.md

```diff
@@ -246,14 +246,14 @@ The following list defines Tizen Native API. The list describes the API modules
-  | Voice control engine  | Provides functions to operate Voice-Control Engine.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0) | [Voice Control](text-input/voice-control-engine.md) |
+  | Multi assistance      | Provides functions for supporting users to use several assistants. | [mobile](../api/mobile/latest/group__CAPI__UIX__MULTI__ASSISTANT__MODULE.html) (since 5.0), [wearable](../api/wearable/latest/group__CAPI__UIX__MULTI__ASSISTANT__MODULE.html) (since 5.0) | [Speech and Voice](speech/overview.md)  |
+  | STT                   | Provides functions to recognize speech.      | [mobile](../api/mobile/latest/group__CAPI__UIX__STT__MODULE.html), [wearable](../api/wearable/latest/group__CAPI__UIX__STT__MODULE.html) | [Speech-to-text](speech/stt.md) |
+  | STT Engine            | Provides functions for operating the Speech-To-Text engine. | [mobile](../api/mobile/latest/group__CAPI__UIX__STTE__MODULE.html) (since 3.0), [wearable](../api/wearable/latest/group__CAPI__UIX__STTE__MODULE.html) (since 3.0) | [Speech-to-text](speech/stt.md) |
+  | TTS                   | Provides functions for synthesizing voice from text and playing synthesized sound data. | [mobile](../api/mobile/latest/group__CAPI__UIX__TTS__MODULE.html), [wearable](../api/wearable/latest/group__CAPI__UIX__TTS__MODULE.html) | [Text-to-speech](speech/tts.md) |
+  | TTS Engine            | Provides functions for operating the Text-To-Speech engine. | [mobile](../api/mobile/latest/group__CAPI__UIX__TTSE__MODULE.html) (since 3.0), [wearable](../api/wearable/latest/group__CAPI__UIX__TTSE__MODULE.html) (since 3.0) | [Text-to-speech](speech/tts.md) |
+  | Voice control         | Provides functions for registering commands and getting notifications when a registered command is recognized. | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__MODULE.html) (since 2.4), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__MODULE.html) (since 3.0) | [Voice Control](speech/voice-control.md) |
+  | Voice control elementary  | Provides functions to control UI components through voice commands. | [mobile](../api/mobile/latest/group__VOICE__CONTROL__ELEMENTARY__MODULE.html) (since 2.4), [wearable](../api/wearable/latest/group__VOICE__CONTROL__ELEMENTARY__MODULE.html) (since 3.0) | [Voice Control](speech/voice-control.md) |     |
+  | Voice control engine  | Provides functions to operate Voice-Control Engine.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0) | [Voice Control](speech/voice-control-engine.md) |
```

Comment:
```suggestion
  | Voice control engine  | Provides functions to operate Voice-Control Engine.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0) | [Voice Control Engine](speech/voice-control-engine.md) |
```

How about using the title of the page?